### PR TITLE
FEATURED WORK - ISSUE RESOLVED

### LINKED ISSUE #48: FEATURED WORK CORRECTION
RESOLVING THIS PROBLEM 

image

### PR DESCRIPTION
**Pull request by rolikaagarwal**
closes #48 
image

### CODE REVIEW

**Comment by AbhiPatel10** [comment]
You are removing testimonial section, why ?

**Comment by rolikaagarwal** [comment]
I'll resolve this.

File: css/style.css [hunk 1]
```diff
@@ -607,54 +607,54 @@ ul {
...
-  font-size: 0.8rem;
-  line-height: 1.5;
+  font-size: 0.87rem;
+  line-height: 1.6;
   letter-spacing: 0.099rem;
 }
-.whychooseus-coloumn hr {
+.col hr {
   margin-top: 10px;
   width: 30%;
```
Comment (AbhiPatel10) [inline css/style.css:659]: Why you are changing "Why choose us section" ?

Remove your whole changes and just change what you want only.

**Review by AbhiPatel10 — CHANGES_REQUESTED** [review]

**Comment by rolikaagarwal** [comment]
Changes Fixed

**Pull request merged**